bottlecap/src/metrics/enhanced/lambda.rs

```diff
@@ -105,6 +107,65 @@ impl Lambda {
         }
     }
 
+    pub(crate) fn generate_network_enhanced_metrics(
+        network_offset: NetworkData,
+        network_data: NetworkData,
```

Comment:
I wonder if we should rename them to denote what they are, the offset is great, but not sure if `network_data` is descriptive enough

Reply:
Would `network_data_offset`/`network_data_start` and then `network_data_end` be more clear